sdk/go/common/util/gitutil/git.go

```diff
@@ -155,7 +163,7 @@ func TryGetVCSInfo(remoteURL string) (*VCSInfo, error) {
 			project = groups["owner_and_repo"]
 			project = strings.TrimSuffix(project, defaultGitCloudRepositorySuffix)
 		}
-	} else if strings.HasPrefix(remoteURL, "http") {
+	} else if strings.HasPrefix(endpoint.Protocol, "http") {
```

Comment:
Also, this will break support for "https". The `HasPrefix` worked for http: and https:.
For HTTP, NewEndpoint uses the URL scheme as-is.

https://github.com/go-git/go-git/blob/1a1bc149278ecdffe85d8890f5733d02abf58b8f/plumbing/transport/common.go#L231

So this should be `case "http", "https"`.

Reply:
I'm pretty sure this was still working -- you're right the scheme would be "https://" but that still has the "http" prefix.

In any case, the switch is cleaner / more straightforward and it's using that now.

Reply:
Oops, yeah, you're right, it would've still worked (minus the endpoint.Protocol panic).
I somehow erased the "HasPrefix" in the new version as I was reading it.